README.md

```diff
@@ -38,11 +38,42 @@ NOTE: by default the waggle-dance-rpm module will build an RPM artifact for inst
 
 ## Installing
 
-It is intended that Waggle Dance is run as a service serving as a proxy to different metastores. The primary configured metastore is the only one to which you may also write data via Hive. It can either be installed on an existing EMR instance (for example the EMR cluster of your primary metastore) or on a separate EC2 instance.
+Waggle Dance comes in two binaries distributions: TGZ and RPM. It can either be installed on an existing EMR instance (for example the EMR cluster of your primary metastore) or on a separate EC2 instance.
+
+### TGZ package
+
+You can uncompress the file by executing:
+
+    tar -xzf waggle-dance-<version>-bin.tgz
+
+Although it's not necessary, we recommend to export the environment variable _WAGGLE_DANCE_HOME_ by setting its value to the location of the uncompressed directory:
```

Comment:
What happens if you don't set WAGGLE_DANCE_HOME? We should mention somewhere what it would do in this case.